### PR TITLE
SC-5762 custom metrics docs

### PR DESCRIPTION
**Pull request by bsmid**
Added docs that describe how to send custom metrics using influx line protocol.

### CODE REVIEW

File: docs/monitoring/custom-metrics.md [hunk 1]
```diff
@@ -1,394 +1,108 @@
...
-
-``` bash
-curl -v -H 'Content-type: application/json' -d @datapoints.json http://spm-receiver.sematext.com/receiver/custom/receive.json?token=[spm app token]
+In this example we can see:
+- `appStats` - a `measurement` or a `namespace`, it describes the type of metrics in this particular line and it can be
+  anything that makes sense to you. It will be used as a prefix to all metrics in this line (so, full name of a metric
+  `user.requests.count` is actually `appStats.user.requests.count`)
+- `token` - a token of your Sematext monitoring App
+- `os.host` and `region` - two tags used to describe these metrics. You can append more tags by appending them after
+  `region=us` (tags have to be separated by a comma)
```
Comment (otisg) [inline docs/monitoring/custom-metrics.md:40]: This region is unrelated to SC US vs. SC EU?  If so, maybe we could use another tag instead of 'region' to avoid potential confusion.

**Review by otisg — APPROVED** [review]
LGTM.  I'll tweak things a bit after the commit, will be simpler that way.

File: docs/monitoring/custom-metrics.md (hunk 1 above)
Comment (bsmid) [reply docs/monitoring/custom-metrics.md:40]: Yep, I'll change it to... something else.
